src/typescript/frontend/src/lib/utils/allowlist.ts

```diff
@@ -19,7 +19,11 @@ export async function isAllowListed(addressIn: string): Promise<boolean> {
     ? (addressIn as `0x${string}`)
     : (`0x${addressIn}` as const);
 
-  return isInGalxeCampaign(address) || isOnCustomAllowlist(address);
+  const [inCampaign, onAllowlist] = await Promise.all([
+    isInGalxeCampaign(address),
+    isOnCustomAllowlist(address),
+  ]);
+  return inCampaign || onAllowlist;
```

Comment:
Let's avoid doing this in the future.

The whole website is inaccessible for users new users.

This should go in a dedicated PR, and merged ASAP.